### DISCUSSION TITLE
Full translation of Grafana UI to French

### DISCUSSION THREAD

**Discussion by saadnabs** [grafana/grafana#63400] [General]
French customer is using the UI translated in French but has seen a few elements that remained in English, listing these out here for completeness:
- Time picker UI shows all the time options absolute and relative in English

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
